api/tests/phones_views_tests.py

```diff
@@ -1026,6 +1026,26 @@ def test_inbound_call_valid_twilio_signature_disabled_number(
     mocked_twilio_client.messages.create.assert_not_called()
 
 
+def test_inbound_call_valid_twilio_signature_no_remaining_seconds(
```

Comment:
🎉 It is nice to have a test for this functionality!

Reply:
As per updated acceptance criteria on the MPP-2449, we are returning a non-negative integer on `remaining_minutes` completed in b9499ef